src/main/java/io/blockfrost/sdk/api/model/AddressUtxo.java

```diff
@@ -31,11 +31,22 @@ public class AddressUtxo {
     /**
      * Array of TransactionOutputAmount objects
      */
-    private List<TransactionOutputAmount> amount = new ArrayList<TransactionOutputAmount>();
+    private List<TransactionOutputAmount> amount = new ArrayList<>();
     /**
      * Block number of the UTXO
```

Comment:
The doc says `Block hash of the UTXO` for the description of that field, not `number`
I an change that as part of this PR if required